project/LienShark/Readme.rst

```diff
@@ -0,0 +1,129 @@
+=================
+Project
+=================
+EigenApproWithGersgorin
+
+Basic Information
+=================
+
+GitHub repository :https://github.com/LienShark/EigenApproWithGersgorin
+About:
+  First step:
+    I will first try to write the basic spectral decomposition and SVD code to use in PCA, and find out the problems in the field (ML/Finance/Genetic information)
+    where I want to test the effect as a test case, and record the time and results of this approach.
+
+  second step:
+    I will try to use Gersgorin Theorem to find eigenvalue approximation to do spectral,
+    hope can speed up PCA compression as much as possible while sacrificing the least accuracy,
+    then provide APIs for related applications that will be used!
+    In my imagination, if the matrix dimensions to be processed today reach the level of millions of rows and columns,
+    it may be very troublesome and slow to calculate the eigenvalues!
+    So I thought about the matrix theory course I took in college, in which Gersgorin Theorem mentioned 
+    that all eigenvalues ​​will fall within certain specific circles.
+    I thought if I could shrink this circle as much as possible through other methods 
+    (or is not narrowed, but the guess is accurate enough within the range) 
+    I have the opportunity to avoid the tedious calculation of eigenvalues ​​and further do PCA
+
+  Final step:
+    I will compare two method, and analysis it! 
+
+Problem to Solve
+================
+First, I will learn how to do SVD decomposition and spectral decomposition for use in PCA.
+I will also survey the current methods of SVD decomposition and spectral decomposition, compare and improve them.
+second,I hope I still have time to add Gersgorin Theorem's method to compare and see if 
```

Comment:
It is clear to pick a numerical method as a baseline.  You mentioned to use an existing package in [your comment](https://github.com/yungyuc/nsdhw_24au/pull/47#issuecomment-2365136211), but I am not seeing what they are in the proposal.  If you wrote them in the proposal, please leave an inline comment to highlight for review.

It will be hard for you to keep track of items you did not write in the plan.